conda-linux-64.lock

```diff
@@ -0,0 +1,544 @@
+# platform: linux-64
+# env_hash: e3fa4018900c5e40bfe421cb9965ab1af1ed4a3ac2db2f0ff7276c8b5f5fe6ee
+
+@EXPLICIT
+
+https://conda.anaconda.org/conda-forge/linux-64/_libgcc_mutex-0.1-conda_forge.tar.bz2#d7c89558ba9fa0495403155b64376d81
```

Comment:
@ocefpaf one last small question - i think i understand what is. happening here. Is this the exact list of each package that is being installed with the url to the install? 

i'll go ahead and merge this. thank you again.

Reply:
Yes. That is correct. conda-lock splits the solve and the installation so you can solve once (which is slow) and install multiple times (preserving the original solve everywhere). That is good for CIs and Docker images b/c you will always get the exact same environment!

To update the environment you'll need to solve again by running conda-lock and commiting this lock file. Another bonus is that the diff will be much clearer b/c we'll see the exact artifact that changed.

Reply:
ok i think i understand. Thank you. So essentially `conda lock` solves the environment which  will happen locally?  And then you push and the envt can just build without solving again? Am I summarizing what you said correctly?

Thats really great  because dockerhub is slow. i really appreciate the help here. Do you suggest moving to this approach for all of our docker containers? the approach  being using conda lock but also that miniconda / conda forge  base image? i have on other one that i use for our jupyter hub?

Reply:
> ok i think i understand. Thank you. So essentially conda lock solves the environment which will happen locally? And then you push and the envt can just build without solving again? Am I summarizing what you said correctly?

Correct.

> Do you suggest moving to this approach for all of our docker containers? the approach being using conda lock but also that miniconda / conda forge base image? i have on other one that i use for our jupyter hub?

I'm biased but we did create conda-lock for that kind of use :smile: